tests/Unit/API/WP/NotificationsServiceTest.php

```diff
@@ -83,6 +84,7 @@ function ( $value, $name ) {
 		);
 
 		add_filter( 'woocommerce_gla_notifications_enabled', '__return_true' );
+		add_filter( 'woocommerce_gla_is_pull_enabled_for_datatype', '__return_true' );
```

Comment:
As mentioned I think it's better to mock the get for `OptionsInterface::API_PULL_SYNC_MODE` to return a specific sync mode instead of just returning true for all modes.

Reply:
I will wait for clarification in this https://github.com/woocommerce/google-listings-and-ads/pull/2804#discussion_r1979371733 as it may impact the options.